src/resdk/resources/__init__.py

```diff
@@ -54,15 +54,23 @@
    :members:
    :inherited-members:
 
-.. autoclass:: resdk.resources.PredictionValue
+.. autoclass:: resdk.resources.PredictionField
    :members:
    :inherited-members:
 
 .. autoclass:: resdk.resources.PredictionGroup
    :members:
    :inherited-members:
 
-.. autoclass:: resdk.resources.PredictionField
+.. autoclass:: resdk.resources.PredictionPreset
+   :members:
+   :inherited-members:
+
+.. autoclass:: resdk.resources.PredictionValue
+   :members:
+   :inherited-members:
+
+.. autoclass:: resdk.resources.Variant
```

Comment:
Any reason not to include also other Variant classes, e.g. VariantAnnotation, VariantExperiment, etc...?

Reply:
Added.